spreadsheet/formula/context.go

```diff
@@ -46,6 +46,12 @@ type Context interface {
 	// IsDBCS returns if workbook default language is among DBCS.
 	IsDBCS() bool
 
+	// LastColumn returns the name of last column which contains data in range of context sheet's given rows.
```

Comment:
Adding methods to interfaces has some risk of breaking compatibility, but if it is necessary we can keep it.

Reply:
Looks like it is needed, we don't have another way to access sheet info without casting to evalContext (which is also private)

Reply:
OK